.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,61 @@
+name: Run Tests
+
+on: [push, pull_request]
+
+jobs:
+
+  test-library:
+    runs-on: ubuntu-latest
+    container: ghcr.io/quattor/quattor-test-container:latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Install dependencies
+      run: yum -y install subversion
```

Comment:
Seems a bit odd to need this? Does one of the scripts depend on it?

Reply:
`create-vanilla-SCDB` which is used to test the templates does